VisualStudio.gitignore

```diff
@@ -80,6 +80,8 @@ ipch/
 *.opensdf
 *.sdf
 *.cachefile
+*.VC.db
```

Comment:
Do you have an external link which explains the addition of this new file? (and perhaps why it's not under `.vs/` :grin:)